### PR TITLE
add flutter obfuscation support 

### PR DESCRIPTION
**Pull request by eldrig0**

### CODE REVIEW

File: flutter-build/action.yml [hunk 1]
```diff
@@ -50,13 +75,28 @@ runs:
...
+        # 4. search for word that ends with certain strings and then remove the ending . if any and convert to relative path if needed
         artifact_path=$(cat flutter_build.log | awk '{for (word=1;word<=NF;word++) if ($word~/release.apk|release.aab|.xcarchive|.ipa/) print $word}' | sed 's/\.$//')
 
         case $artifact_path in
           /*) echo "artifact-path=$artifact_path" >> $GITHUB_OUTPUT ;;
           *) echo "artifact-path=${{ inputs.working-directory }}/$(echo $artifact_path)" >> $GITHUB_OUTPUT ;;
-        esac
\ No newline at end of file
+        esac
+    - if: ${{ inputs.sentry-project != '' && inputs.obfuscate }}
```
[the remark this answers is not in the document]
Comment (eldrigo) [reply flutter-build/action.yml:94]: tested it yesterday, and it passed, will try again ans see.

File: flutter-build/action.yml [hunk 2]
```diff
@@ -23,6 +23,31 @@ inputs:
...
+  sentry-project:
+    description: 'Needed if obfuscate is enabled while building'
+    default: ''
+    type: string
+    required: false
+  sentry-auth-token:
+    description: 'Needed if obfuscate is enabled while building'
+    type: string
+    default: ''
+    required: false
```
[the remark this answers is not in the document]
Comment (azhaHaap) [reply flutter-build/action.yml]: fixed, removed the type parameter.

File: flutter-build/action.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (azhaHaap) [reply flutter-build/action.yml]: fixed

File: flutter-build/action.yml [hunk 3]
```diff
@@ -23,6 +23,31 @@ inputs:
...
+    description: 'Needed if obfuscate is enabled while building'
+    default: ''
+    type: string
+    required: false
+  sentry-org:
+    description: 'Needed if obfuscate is enabled while building'
+    default: ''
+    type: string
+    required: false
+  sentry-project:
```
[the remark this answers is not in the document]
Comment (azhaHaap) [reply flutter-build/action.yml:38]: I changed the description but want to keep the input name consistant with the name in Org secrets.

**Pull request closed without merging**